File: Iniciante/URI 1015.hs
import Text.Printf

main = do
    line1 <- getLine
    line2 <- getLine

    let (x1:y1:_) = map (read :: String -> Double) $ words line1
    let (x2:y2:_) = map (read :: String -> Double) $ words line2
    let distancia = sqrt(((x2 - x1) ** 2.0) + ((y2 - y1) ** 2.0))

    putStrLn $ printf "%.4f" distancia

File: Iniciante/URI 1015.php
<?php
    $line1 = explode(" ", trim(fgets(STDIN)));
    $x1 = floatval($line1[0]);
    $y1 = floatval($line1[1]);

    $line2 = explode(" ", trim(fgets(STDIN)));
    $x2 = floatval($line2[0]);
    $y2 = floatval($line2[1]);

    $distancia = sqrt(pow(($x2 - $x1), 2.0) + pow(($y2 - $y1), 2.0));
    $distancia = number_format($distancia, 4, ".", "");
    echo $distancia . "\n";
?>